### ISSUE TITLE
unable to request new issue

### ISSUE THREAD

**Issue by jitendraauxesis** [blockchain-certificates/cert-issuer#79]
I requested digital certificate using cert-viewer repository but Im unable to issue the certificate, and it returns **"Unhandled exception: localhost:27017: [Errno 111] Connection refused"** on **http://localhost:5000/request** .
Suggest me what change I need to correct it.

**Comment by jitendraauxesis** [comment]
I have created certificate.json using cert-tools git repository. When I copied  this certificate.json  to  /cert-issuer/data/unsigned_certificates and I tried to issue using cert-issuer in docker container all steps working fine. But when I opening certificate.json in cert-viewer it wont opening to verify it. What something is wrong? After clicking on certificate.json in cert-viewer it returns **"Unhandled exception: Unknown string format"**.
